sanic/request.py

```diff
@@ -158,13 +158,17 @@ def raw_args(self):
     def cookies(self):
         if self._cookies is None:
             cookie = self.headers.get('Cookie')
+            self._cookies = {}
             if cookie is not None:
-                cookies = SimpleCookie()
-                cookies.load(cookie)
-                self._cookies = {name: cookie.value
-                                 for name, cookie in cookies.items()}
-            else:
-                self._cookies = {}
+                for chunk in cookie.split(';'):
```

Comment:
I'm worried this logic is going to be significantly slower than whatever was happening before.

Reply:
I see. It's depended on your goals and viewpoint. Do you see it as a bug? :/ or you are accepting it to increase performance?! :/
(I didn't run benchmark on it and I don't know how much this code increases or decreases performance)

Reply:
yep, we might just follow the builtin lib behavior for now and keep an eye on that.

Reply:
If you want to follow _http.cookie_ Please mention it and it's problems in your documentation :/